targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/TARGET_NUCLEO_F429ZI/system_stm32f4xx.c

```diff
@@ -187,10 +187,6 @@ void SetSysClock(void);
   */
 void SystemInit(void)
 {
-  /* FPU settings ------------------------------------------------------------*/
-  #if (__FPU_PRESENT == 1) && (__FPU_USED == 1)
-    SCB->CPACR |= ((3UL << 10*2)|(3UL << 11*2));  /* set CP10 and CP11 Full Access */
-  #endif
```

Comment:
Hi, NUCLEO_F429ZI shares some files with DISCO_F429ZI, including startup files. 
Since integration of uvisor for DISCO_F429ZI, this line of the SystemInit is copied into a new function (System_InitPre) that is called before the uvisor_init (if present) and system_init in the startup files.
Then this line is not removed but moved to https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F429xI/device/system_init_pre.c